examples/sort.cu

```diff
@@ -41,7 +41,7 @@ void initialize(thrust::device_vector<int>& v1, thrust::device_vector<int>& v2)
   for(size_t i = 0; i < v1.size(); i++)
   {
     v1[i] = dist(rng);
-    v2[i] = i;
+    v2[i] = static_cast<int>(i);
```

Comment:
I would just make `i` be `size_t` to avoid a `static_cast` in example code.

Reply:
Oh, then you'd have to `static_cast` the size variable. Maybe just change the `device_vector` to be `size_t` not `int`.

Reply:
That would require refactoring the entire example.

Any real C++ application will require some `static_cast` in these sorts of situations, it's not unexpected or easily avoidable. I'm going to leave this as is.